bin/load-mocks

```diff
@@ -66,9 +67,13 @@ def make_sentence(words=None):
 
 
 def create_sample_event(*args, **kwargs):
-    event = _create_sample_event(*args, **kwargs)
-    features.record(event)
-    return event
+    try:
+        event = _create_sample_event(*args, **kwargs)
+        features.record(event)
```

Comment:
since i think the `event = _create_sample_event(*args, **kwargs)` is the only thing that will raise a `HashDiscarded`, can you put the rest into an else block? so something like:

```python
try:
    event = _create_sample_event(*args, **kwargs)
except HashDiscarded as e:
    print "> Skipping Event: {}".format(e.message)
else:
    features.record(event)
    return event
```